src/Core/SettingsChangesHistory.cpp

```diff
@@ -766,6 +766,7 @@ const VersionToSettingsChangesMap & getMergeTreeSettingsChangesHistory()
         addSettingsChanges(merge_tree_settings_changes_history, "25.6",
         {
             {"cache_populated_by_fetch_filename_regexp", "", "", "New setting"},
+            {"allow_coalescing_columns_in_partition_or_order_key", true, false, "New setting to allow coalescing of partition or sorting key columns."},
```

Comment:
Why is this enabled for old compatibility values if the engine is new? Shouldn't it be the same for both?

Reply:
My bad. Thanks for fixing it.